content/basics/use-ably.textile

```diff
@@ -0,0 +1,139 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the options available to use with Ably"
+---
+
+Ably provides a number of APIs, SDKs, and adaptors and this page will briefly cover these options available to help you integrate with Ably in the way best suited for you and your business. 
+
+h2(#sdks). Ably SDKs with Realtime and REST Interfaces
+
+All official client library SDKs developed by Ably offer a consistent yet idiomatic API across every language. So regardless of your development environment or language, Ably keeps things simple for you by being consistent and predictable.
```

Comment:
We should be consistent here with terminology - I'd suggest sticking with "Ably SDKs" as per the title. This paragraph also has a bit of a marketing spin to it, whereas this should be informational in terms of what our SDKs do and the difference between the two interfaces.